apps/calendar/js/views/view_event.js

```diff
@@ -27,6 +30,13 @@ ViewEvent.prototype = {
 
   _initEvents: function() {
     EventBase.prototype._initEvents.apply(this, arguments);
+    this.getEl('description').addEventListener('click', this);
```

Comment:
I think you want to listen in `onactive` and then remove the listener in `oninactive`